components/config/prefs.rs

```diff
@@ -198,6 +198,24 @@ mod gen {
                 #[serde(rename = "fonts.default-monospace-size")]
                 default_monospace_size: i64,
             },
+            threadpools: {
+                image_cache_workers: {
+                    fallback: i64,
```

Comment:
We should docement all pref. This custom macro allow doc comment on sturct fields which are `fallback` and `max`

Reply:
done